src/partition/growfs.c

```diff
@@ -25,13 +24,15 @@
 #include "parse-util.h"
 #include "pretty-print.h"
 #include "resize-fs.h"
+#include "sd-device.h"
```

Comment:
Please move `sd-device.h` earlier place. See our [coding style](https://github.com/systemd/systemd/blob/e109541fe6988fd0e73b92226f36f41abc829883/docs/CODING_STYLE.md?plain=1#L124-L135) and other files which include sd-device.h:
```
#include <...>
#include <...>

#include "sd-device.h"

#include "blockdev-util.h"
#include "btrfs-util.h"
...
```